x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/chart_switch/index.tsx

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './chart_switch';
```

Comment:
```suggestion
export { ChartSwitch } from './chart_switch';
```

Let's export only the component that is being imported in other places and not everything